src/server/json_family.cc

```diff
@@ -326,8 +420,80 @@ OpResult<string> OpDoubleArithmetic(const OpArgs& op_args, string_view key, stri
   return output.as_string();
 }
 
+OpResult<long> OpDel(const OpArgs& op_args, string_view key, string_view path) {
+  long total_deletions = 0;
+  if (path.empty()) {
+    auto& db_slice = op_args.shard->db_slice();
+    auto fres = db_slice.FindExt(op_args.db_ind, key);
```

Comment:
DbSlice has `Del` function that does what you do here.

Reply:
How do I call to `DbSlice::Del` without supplying in the first argument a `PrimeIterator`?

Reply:
I did not understand the question.
btw, you do not have to check for validity of fres.first - Del does it for you.
Also, I find `auto [it, _] = db_slice.FindExt(op_args.db_ind, key);` a bit more readable than using the pair.